components/director/internal/domain/package/fixtures_test.go

```diff
@@ -364,30 +363,21 @@ func fixBasicInputSchema() *graphql.JSONSchema {
 	return &jsonSchema
 }
 
-func fixBasicSchema(t *testing.T) *interface{} {
-	sch := fixBasicInputSchema()
-	require.NotNil(t, sch)
-	var obj map[string]interface{}
-
-	err := json.Unmarshal([]byte(*sch), &obj)
-	require.NoError(t, err)
-	var objTemp interface{}
-	objTemp = obj
-	return &objTemp
+func fixBasicSchema() *string {
```

Comment:
Maybe we should validate `fixSchema`, in case that somebody change it.

Reply:
Looks like this schema isn't even parsed anywhere in this domain, I don't think validation is really necessary.